apps/agent/src/main/tray.ts

```
@@ -116,6 +140,29 @@ class TrayMenu {
 
 	private initTray() {
 		this.tray = new Tray(this.nativeIconPath());
+		this.tray.setTitle('State: Startup', {
+			fontType: 'monospacedDigit'
+		});
+		this.tray.setToolTip('Agent is starting up');
+	}
+
+	public updateStatus(
+		menuId: 'keyboard_mouse' | 'network' | 'afk',
+		checked: boolean = false
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)